### PR TITLE
Feature/add squad page

### PR DESCRIPTION
**Pull request by str1nker**

### CODE REVIEW

**Comment by ARtoriouSs** [comment]
Lets move squad number a little down and right to be in the center

image

*Edit:* when I opened devtools additional information comes to center and it's ok, probably this is screen size issue

image

**Comment by ARtoriouSs** [comment]
On mobile view commanders and deputy commanders shifted, probably we can move role text on the top of member card

image

**Comment by ARtoriouSs** [comment]
Also lets remove this sorts:

image

We can just put commander, deputys and journalist to the top, and below will be students in order of duty like [here](https://aginvlad.github.io/VzvodApp/)

**Comment by str1nker** [comment]
> On mobile view commanders and deputy commanders shifted, probably we can move role text on the top of member card
> 
> image

@ARtoriouSs  reload the page

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -27,33 +26,43 @@ function logout(reducer) {
 }
 
 export default function Navbar() {
-  const classes = useStyles();
```
Comment (ARtoriouSs) [inline src/components/Navbar.jsx:30]: I think we need semicolons here and below

**Review by ARtoriouSs — CHANGES_REQUESTED** [review]

**Review by ARtoriouSs — APPROVED** [review]

**Comment by ARtoriouSs** [comment]
nice!

**Pull request merged**